supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -112,6 +112,7 @@ Ensure that user-replaced values have the following characteristics:
 * Lowercase, unless the rest of the related text is uppercase or another capitalization scheme
 * Italicized
 * If the user-replaced value is referencing a value in code or in a command that is normally monospace, also use monospace for the user-replaced value
+* If you want to use a user-replaced value in example output, format the replaceable value with italics and in angle brackets. Alternatively, if you choose to use an example value instead, do not italicize the example value or place it in angle brackets.
```

Comment:
```suggestion
* If you want to use a user-replaced value in example output, format the replaceable value with italics and in angle brackets. Alternatively, if you choose to use an example value instead, do not italicize the example value or place it in angle brackets.
```
That last sentence is a bit ambiguous to my ears. Not sure if it means "do not italicize the example value **and** do not place it in angle brackets" or does it mean "do not italicize the example value. Or, alternatively, place it in angle brackets."

Reply:
Thanks, Brian -- I take your point about ambiguity. I'm pretty sure I meant `do not italicize the example value and do not place it in angle brackets` and your suggestion looks good to me.